database.php

```diff
@@ -1,18 +1,38 @@
 <?php
+function getDb(){
+    $db = new PDO ('mysql:host=db; dbname=kimberley-collection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
 
-$db = new PDO ('mysql:host=db; dbname=kimberley-collection', 'root', 'password');
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-$query = $db->prepare("SELECT `name`,`main-colour`,`category`,`material`,`size`,`heavyness`,`img-url` FROM `kimberley-collection`;");
+function retrievePaperWeights($db){
+    $query = $db->prepare("SELECT `name`,`main-colour`,`category`,`material`,`size`,`heavyness`,`img-url` FROM `kimberley-collection`;");
 
-$query->execute();
+    $query->execute();
 
-$outputs = $query->fetchAll();
+    return $query->fetchAll();
 
-//echo '<pre>';
-//var_dump($outputs);
-//echo '</pre>';
+}
+
+function generateHtml(array $paperweights) : string
+{
```

Comment:
you might want to move this up a line for consistency with the rest of your code